libpod/container_internal_linux.go

```diff
@@ -355,6 +360,10 @@ func (c *Container) checkpoint(ctx context.Context, keep bool) (err error) {
 
 func (c *Container) restore(ctx context.Context, keep bool) (err error) {
 
+	if !criu.CheckForCriu() {
+		return errors.Errorf("checkpointing a container requires at least CRIU 3.11")
```

Comment:
Change to "restoring a container" instead of "checkpointing a container"

Reply:
Ah, copy/paste. Will fix.